src/pages/main-page/main-page.tsx

```diff
@@ -0,0 +1,283 @@
+import React from 'react';
+
+type MainPageProps = {
+	totalPlaces: number;
+	favoriteCount: number;
+};
+
+/**
+ * Компонент главного экрана
+ * @component
+ * @param placesCount общее количество найденный мест
+ * @param favoriteCount Количество избранных
+ */
```

Comment:
Молодец что пишешь JSDoc, очень круто: 
+ но имя параметра сейчас отличается. 
+ в тс есть уже более удобное где можно оисать их
```suggestion
type MainPageProps = {
	/** общее количество найденный мест */
	totalPlaces: number;
	/** Количество избранных */
	favoriteCount: number;
};

/**
 * Компонент главного экрана
 */
 ```